managers/oclc_auth.py

```diff
@@ -18,22 +17,20 @@ class OCLCAuthManager:
     
 
     @classmethod
-    def getToken(cls):
+    def get_token(cls):
```

Comment:
The get_token method failed the integration test with an AssertionError when I ran the test:

def test_get_token():
        token = OCLCAuthManager.get_token()
    
         assert token != None
E assert None != None

Reply:
Sent over the oclc creds!

Reply:
The test passes now